proto/zetachain/zetacore/crosschain/cross_chain_tx.proto

```diff
@@ -135,4 +135,5 @@ message CrossChainTx {
   repeated OutboundParams outbound_params = 10;
   ProtocolContractVersion protocol_contract_version = 11;
   RevertOptions revert_options = 12 [ (gogoproto.nullable) = false ];
+  string error = 13;
```

Comment:
I think we should avoid changing the cctx object, 
If the change is absolutely necessary, we should group this into 

https://github.com/zeta-chain/node/issues/1949

The bigger task would be part of a future sprint , and we make all necessary changes to the object , and it would be easier to review . 

Would also be great to hear you thoughts on this @lumtis

Reply:
I did some quick testing; for cctx older before the upgrade , we would return empty for this field, Which seems okay. 

We should certify as part of this pr that we wont face issues similar to  
https://github.com/zeta-chain/node/issues/2835

Reply:
This is fine if asked explicitely by some partners
https://github.com/zeta-chain/node/issues/1949 will break compatibility, and will be pushed back for some time for now
I think adding a new field will not break anything

Reply:
Message moved to cctx.status.error_message